recipe/windows_compat.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/include/X11/Xlibint.h b/include/X11/Xlibint.h
+index 50099b4..4b388f4 100644
+--- a/include/X11/Xlibint.h
++++ b/include/X11/Xlibint.h
+@@ -45,6 +45,7 @@ from The Open Group.
+ 
+ #ifdef XTHREADS
+ #include <X11/Xthreads.h>
++#undef CreateWindow /* conda-forge: work around bug in Xwindows.h exposed here */
```

Comment:
is this safe for all operating systems? or do you have to add a windows guard?

Reply:
If `CreateWindow` hasn't been `#define`-d, it's a no-op, and if it has been, well, that's what we need to fix. So it's fine not to have a Windows guard here as far as I can see.